receiver/hostmetricsreceiver/internal/scraper/processscraper/process_metadata_benchmark_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build windows
+
+package processscraper
```

Comment:
This file could be removed when we remove the featureflag to fallback to the old behavior.